src/js/plugins/carousel.js

```diff
@@ -34,6 +34,10 @@ const CONFIG_DEFAULT = {
     pageX: 'Vai a pagina %s',
     play: 'Attiva autoplay',
     pause: 'Pausa autoplay',
+    carousel: 'carosello',
```

Comment:
```suggestion
    carousel: 'Carosello',
```

Non sono comunque sicuro che "Carosello" sia un termine comprensibile in italiano.